_posts/2018-04-11-dont-make-users-think-about-updating-software.md

```diff
@@ -11,35 +11,35 @@ tags:
   - update-service
 ---
 
-As software developers, we want the people using the software we create to ~~enjoy~~ not _hate_ the experience. Developers strive to create a positive user experience to increase the popularity and usage of their software. Consequently, many hours are spent debating and designing user interactions that are simple, effective, and intuitive.
+As software developers, we want the people who are using our software to ~~enjoy~~ _not hate_ the experience. Developers strive to create a positive user experience to increase the popularity and usage of their software. Consequently, we spend many hours designing, debating, and redesigning to ensure that user interactions with our software are simple, effective, and intuitive.
 
 ### skip to the [tl;dr <i class="fas fa-long-arrow-alt-down"></i>](#tldr)
 
-> The people that are using your software are motivated by the desire to save time
+> The people who are using your software are motivated by the desire to save time.
```

Comment:
These subheading lines (whatever you want to call them) are complete sentences, so I went ahead and added ending punctuation.